contracts/main/stablecoin-core/adapters/FarmableTokenAdapter/AnkrCollateralAdapter.sol

```diff
@@ -337,16 +337,15 @@ contract AnkrCollateralAdapter is IFarmableTokenAdapter, PausableUpgradeable, Re
   ) internal onlyCollateralManager {
         uint256 certsToMoveRatio;
         if(_share >= stake[_source]){
-            uint256 CertsMove = recordRatioNCerts[_source].CertsAmount;
-            recordRatioNCerts[_source].CertsAmount = 0;
-            recordRatioNCerts[_destination].CertsAmount = CertsMove;
+            certsToMoveRatio = WAD;
         } else {
             require(_share < stake[_source], "AnkrCollateralAdapter/tooMuchShare");
             certsToMoveRatio = wdiv(_share, stake[_source]);
```

Comment:
Shall we handle the condition if stake[_source] is zero before division.

Reply:
thanks for the review, require statement added to check if stake[_source] is zero